libnmstate/nm/team.py

```diff
@@ -82,17 +82,8 @@ def get_info(device):
     """
```

Comment:
Is the bugzilla still relevant?

Reply:
You mean https://bugzilla.redhat.com/show_bug.cgi?id=1798947 ?

Reply:
Never mind. You mean 1792232.